portfolio/src/main/java/com/google/sps/servlets/DateTimeServlet.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+
+/** Servlet that responds with the current date. */
+@WebServlet("/dateTime")
+public class DateTimeServlet extends HttpServlet {
+
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("uuuu/MM/dd");
+    LocalDate localDate = LocalDate.now();
```

Comment:
The `LocalDate localDate = LocalDate.now();` and `LocalTime localTime = LocalTime.now();` can be done inside the doGet. That way the date and time can be an updated time everytime the get is called.

Reply:
Done in my last commit 👍